src/Symbol/Loader/FileSymbolLoader.php

```diff
@@ -34,8 +34,16 @@ public function load(PackageInterface $package): Generator
         $paths = [];
 
         foreach ($this->autoloadTypes as $autoloadType) {
-            /** @var array<string, string> $autoloadPaths */
-            $autoloadPaths = $package->getAutoload()[$autoloadType] ?? [];
+            /**
+             * @var array{
+             *      psr-0?: array<string, string|string[]>,
+             *      psr-4?: array<string, string|string[]>,
+             *      classmap?: list<string>,
+             *      files?: list<string>
+             * }
+             */
```

Comment:
This array shape definition is also not needed, as its interfered with the `getAutoload` from `PackageInterface`
Also not needed with mentioned change of `normalizePsrStructure`